src/main/kotlin/org/jitsi/jibri/metrics/JibriMetrics.kt

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright @ 2024-Present 8x8, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.jitsi.jibri.metrics
+
+import com.timgroup.statsd.NonBlockingStatsDClient
+import org.jitsi.jibri.RecordingSinkType
+import org.jitsi.jibri.status.ComponentBusyStatus
+import org.jitsi.jibri.status.ComponentHealthStatus
+import org.jitsi.jibri.status.JibriStatus
+import org.jitsi.utils.logging2.createLogger
+
+class JibriMetrics {
+    private val logger = createLogger()
+
+    private val statsd = if (StatsConfig.enableStatsD) {
+        NonBlockingStatsDClient(
+            "jibri",
+            StatsConfig.statsdHost,
+            StatsConfig.statsdPort
+        )
+    } else {
+        null
+    }
+
+    private fun incrementStatsDCounter(aspect: String, vararg tags: String) {
```

Comment:
Does this ever actually take more than one tag String?

I see places that pass "tags: String" but that's still just a single String.